R/utm_dd.R

```diff
@@ -30,25 +30,31 @@
 #' @examples \dontrun{
 #'
 #'}
-utm_dd <- function(zone=NULL, easting=NULL, northing=NULL, datum="NAD83", data=NULL, key=NULL) {
+utm_dd <- function(zone = NULL, easting = NULL, northing = NULL, 
+                   datum = "NAD83", data = NULL, key = NULL) {
+  
+  if (!requireNamespace("rgdal") || !requireNamespace("sp")) {
+    stop("You need the 'sp' and 'rgdal' packages installed to use this function")
+  }
 
   get_dd <- function(d) {
     # TODO Vectorise by grouping input by datum and zone
     # requires a one-row dataframe with zone, easting, northing, datum (in that order)
     
-    utm <- SpatialPoints(d[2:3], proj4string=CRS(paste0("+proj=utm +datum=", d[4], " +zone=", d[1])))
-    sp <- spTransform(utm, CRS("+proj=longlat"))  
+    utm <- sp::SpatialPoints(d[2:3], 
+                             proj4string = CRS(paste0("+proj=utm +datum=", d[4], " +zone=", d[1])))
```

Comment:
I think we need `sp::CRS` here and `sp::coordinates` below

Reply:
Yup, I missed that. I also missed the @imports that called rgdal and sp, which is why I had trouble with the NAMESPACE. Fixed now in fee8dc9.

Reply:
👍